doc/contributing/releases.md

```diff
@@ -966,6 +966,13 @@ a `NODEJS_RELEASE_HOST` environment variable:
 NODEJS_RELEASE_HOST=proxy.xyz ./tools/release.sh
 ```
 
+In case `gpg` is unable to autoselect a key, you can retry using the
```

Comment:
How can we do it with git node release prepare?

Reply:
as far as I can tell, based on today's run, git node release promote default behavior will just stop and ask you to run it in a separate shell, like this:

```
✔ Ready to continue? yes
   ℹ  You are running in dry-run mode, meaning NCU will not run the `git push` commands, you would need to copy-paste the following command in another terminal window. Alternatively, pass `--run` flag to ask NCU to run the command for you (might not work if you need to type a passphrase to push to the remote).
   ℹ  Run the following command to sign and promote the release:
   ℹ  ./tools/release.sh -i <keyPath>
```

Reply:
I believe the autodetect feature needs to be 100% in case you want to run it in `--run` mode, otherwise we should put extra validation along the way to catch it early on in case values like these are unable to be retrieved

Reply:
That makes sense